src/main/java/com/github/jchartier/slackbot/pomodoro/PomodoroCommandManager.java

```diff
@@ -49,6 +56,12 @@ public void register() {
                 return;
             }
 
+            if (isListCommand(message)) {
```

Comment:
Wouldn't it be better to use a switch case? We could before that distinguish the command from the parameters by splitting on spaces.